File: C++/simplecalculator.cpp
#include <iostream>
using namespace std;

int main() {
    char op;
    double a, b;
    cout << "Enter an operator (+ - * /): ";
    cin >> op;
    cout << "Enter two numbers: ";
    cin >> a >> b;

    switch(op) {
        case '+': cout << "Result: " << a + b; break;
        case '-': cout << "Result: " << a - b; break;
        case '*': cout << "Result: " << a * b; break;
        case '/':
            if (b != 0) cout << "Result: " << a / b;
            else cout << "Cannot divide by zero";
            break;
        default: cout << "Invalid operator";
    }
    return 0;
}

File: simplecalculator.py
def add(x, y): return x + y
def subtract(x, y): return x - y
def multiply(x, y): return x * y
def divide(x, y): return x / y if y != 0 else "Cannot divide by zero"

print("Select operation: + - * /")
op = input("Enter operation: ")

a = float(input("Enter first number: "))
b = float(input("Enter second number: "))

if op == '+':
    print("Result:", add(a, b))
elif op == '-':
    print("Result:", subtract(a, b))
elif op == '*':
    print("Result:", multiply(a, b))
elif op == '/':
    print("Result:", divide(a, b))
else:
    print("Invalid operation.")
